### ISSUE TITLE
Support DiagnosticSource for Entity Framework Core and SQLite

### ISSUE THREAD

**Issue by caiofbpa** [getsentry/sentry-dotnet#1577]
Most Xamarin apps use Entity Framework Core or some sort of SQLite bindings to write local data, and it would be interesting to get performance information on those transactions inside Sentry, given that's already happening with .NETStandard and .NETFramework projects.

Perhaps this is a feature to be considered after the move to MAUI?

**Comment by caiofbpa** [comment]
Yes, we're building apps with Xamarin (without Xamarin.Forms), so by the new nomenclature I think it's going to be ".NET 6 Android" and ".NET 6 iOS"... Having some kind of performance monitoring out of the box for Sentry on those platforms would be really nice, and I guess it would benefit MAUI apps as well.

**Comment by SimonCropp** [comment]
how is this feature different from the existing DiagnosticSource Integration? https://docs.sentry.io/platforms/dotnet/guides/aspnetcore/performance/instrumentation/automatic-instrumentation/#diagnosticsource-integration

**Comment by SimonCropp** [comment]
@caiofbpa does the above DiagnosticSource Integration meet your requirements?
